src/Core/src/Platform/iOS/SwitchExtensions.cs

```diff
@@ -51,14 +51,23 @@ public static void UpdateThumbColor(this UISwitch uiSwitch, ISwitch view)
 		internal static UIView? GetTrackSubview(this UISwitch uISwitch)
 		{
 			if (OperatingSystem.IsIOSVersionAtLeast(13) || OperatingSystem.IsTvOSVersionAtLeast(13))
-				return uISwitch.Subviews?.FirstOrDefault()?.Subviews?.FirstOrDefault();
+				return uISwitch.Subviews?.FirstOrDefaultNoLinq()?.Subviews?.FirstOrDefaultNoLinq();
 			else
-				return uISwitch.Subviews?.FirstOrDefault()?.Subviews?.FirstOrDefault()?.Subviews?.FirstOrDefault();
+				return uISwitch.Subviews?.FirstOrDefaultNoLinq()?.Subviews?.FirstOrDefaultNoLinq()?.Subviews?.FirstOrDefaultNoLinq();
 		}
 
 		internal static UIColor? GetTrackColor(this UISwitch uISwitch)
 		{
 			return uISwitch.GetTrackSubview()?.BackgroundColor;
 		}
+
+		internal static T? FirstOrDefaultNoLinq<T>(this T[] items)
```

Comment:
can you move this into ArrayExtensions?

Reply:
@PureWeen When I move this to ArrayExtensions as is, I get this error: "Error CS0518: Predefined type 'System.Index' is not defined or imported(CS0518) (Core)". I'm not sure why as it appears that this is an error I could get prior to having C#8 but we are in the same project. 

I changed the implementation to something that I don't believe is worse, but let me know if there is a better solution to this?